### ISSUE TITLE
New walk though as the layout has changed

### ISSUE THREAD

**Issue by delboydell** [dale3h/alexa-smartstart#7]
Hello can anyone do a new walk through as it looks like things have changed. Been trying for over a week now and fail at the copy and paste section after all the code has been genarated. I have sent a donation  for server upkeep

**Comment by boomhauer2** [comment]
I am wondering the same! Anyone done this lately?

**Comment by delboydell** [comment]
I am glad I am not alone. I did report an issue with the sever just over a week ago and got as near instant response, perhaps dale3h is on holiday? If you manage to get anywhere in the mean time please let me know, as I am pulling out what little hair I have trying to work it out.

**Comment by goconfigure** [comment]
I am also very interested in an updated walk through. Im happy to donate to the server as well

**Comment by j-madrone** [comment]
I just enabled this today. Steps are almost the same as what is described in the readme. The major differences are:

- The skill ID is not displayed in the upper right corner. You have to click the 'Your Skills' link and then click the Show Skill ID link to get yours.
- Copy/pasting the Intent Schema is just terrible. I manually added all the intents directly in the UI and added all the utterances listed in the config page for the heroku app
- I also had to click the Build Model button in the developer UI, which took a little while. Once I did that, I was able to get Alexa to start and stop my car. She responds in a funny way: Me: 'Alexa, tell Viper to start my car', Alexa: 'Start your engines'

**Comment by goconfigure** [comment]
I have been trying to do this for weeks now. I would love more detail on this project. Like how do you manually add the intents directly. When I cut and paste it says it's over the limit for intents. I was able to get the skill ID and enter my password into the server to get my code.

**Comment by goconfigure** [comment]
I'm not sure how to add the "intents" either.  I'd throw a $20 somebody's way if they could sort this out for me

**Comment by boomhauer2** [comment]
Me to still having troubles I'd throw some cash as well for an exact walkthrough

**Comment by goconfigure** [comment]
Sample utterances syntax is confusing. It is NOT just cut and paste from the herokuapp. The Alexa Dev interface won't even build if you cut and paste the utterances. I would benefit from a screen shot of your build from the Alexa Dev console.

**Comment by goconfigure** [comment]
So if: StartIntent   start the car

is the only command I want to use what is the syntax? I've tried: {StartIntent} {start} the {car} 
StartIntent  {start the car}
{Start}Intent  {start} the {car}

It's driving me just a little crazy :)

**Comment by j-madrone** [comment]
Hey all, I took a shot at rewriting the instructions with screenshots. Let me know if it works for you. https://github.com/jaspersardonicus/alexa-smartstart/tree/patch-1

I also opened a PR, so we will see if anything happens there. 

If you want to donate, my paypal is jasper.sardonicus@gmail.com 😄

**Comment by delboydell** [comment]
@jaspersardonicus I think I am nearly there. Followed your instructions, and in the dev.console I can test the skill and I get the message ladies and gents start your engines. (no response from the car though)
but on the aleax app I get hmmm don't think I know that one for any command relating to the smart start. ??

**Comment by j-madrone** [comment]
@delboydell what did you say to alexa? I used `Alexa, tell viper to start my car`

Have you confirmed the skill is active in your app?

**Comment by delboydell** [comment]
yes the skill is active on the app and I have used Alexa, tell viper to start my car and I get the sorry I don't know that. but typing it in the developer console test field returns the ladies and gents start your engines
I will delete the skill tonight and start again

**Comment by delboydell** [comment]
@jaspersardonicus  I have just deleted the skill and entered it again, including a new  Viper SmartStart Alexa Skill Setup code.
Again in the developer console I can type commands and they get the required responses (no action with the car though)..
I am in the U.K . I don't know if that matters? I have also just added 
◦Copy and paste the Skill URL from the Viper SmartStart Alexa Skill Setup page into the text field under Europe 
too see if that made any difference. ....it didn't.
the newly created skill is enabled on my phone, but for every command I get "I don't know that" .

anyone have any idea where its gone wrong?

**Comment by j-madrone** [comment]
Interesting. I am not too familiar with Alexa skills myself, as this was the first one I created too. Good call entering the Skill URL in the Europe box, too bad it didn't work. You may end up having to reach out to Amazon directly for support there.

**Comment by delboydell** [comment]
Ok I got this working !!!!! deleted the skill and  started again.  The only thing I did different this time was to select "English UK" as default language. in the test skill field I typed alexa start my car with viper , got the ladies and gents, and my lcd remote vibrated and my car started.!!!
So I am looking at changing the wording to make it a little easer to use and change the responses.
Thank you @jaspersardonicus I have sent you a beer via paypal.

**Comment by j-madrone** [comment]
Congratulations! And thanks!

**Comment by j-madrone** [comment]
@delboydell I updated my branch to include the tip about the language selection :+1:

**Comment by nautcasprt6** [comment]
Both of my cars have a SmartStart, but only my 1st car starts. How can I get my second car to start?

**Comment by zbowman** [comment]
> Both of my cars have a SmartStart, but only my 1st car starts. How can I get my second car to start?

Based on how this is setup, you'll need to have a skill set up for each car.
